concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -299,8 +306,11 @@ def test_provider_send_force_payment_with_no_value_to_be_paid_concent_should_rej
         )
 
         with freeze_time("2018-02-05 12:00:20"):
-            with mock.patch('core.message_handlers.base.get_list_of_transactions', _get_requestor_valid_list_of_transactions):
-                with mock.patch('core.message_handlers.base.payment_summary', _get_payment_summary_negative):
+            with mock.patch('core.message_handlers.base.get_number_of_eth_block', self._get_number_of_eth_block):
+                fake_responses = [mock.Mock(), mock.Mock()]
```

Comment:
I think these two shouldn't be created, since you are overriding them in lines below

Reply:
Done